src/sql/workbench/services/tableDesigner/browser/tableDesignerComponentInput.ts

```diff
@@ -580,4 +607,13 @@ export class TableDesignerComponentInput implements DesignerComponentInput {
 			this._viewModel[property] = {};
 		}
 	}
+
+	private createEditActionEvent(edit: DesignerEdit): ITelemetryEvent {
+		const editAction = this._adsTelemetryService.createActionEvent(TelemetryView.TableDesigner, edit.type).withAdditionalProperties({
+			objectType: edit.path[0],
```

Comment:
This isn't going to have any sort of object names in it right?

Reply:
no, just the type